components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -410,6 +410,10 @@ var nodeProps = `
 	"deprecations_count": {
 		"type": "integer",
 		"null_value": 0
+	},
```

Comment:
Adding the created date to be able to count how many nodes existed at a past time period.